src/store/facilityList/types.ts

```diff
@@ -0,0 +1,18 @@
+export interface facility {
+  capacity: number;
+  currentCapacity: number;
+}
+
+export interface FacilityList {
+  cap: number;
+  size: number;
+}
```

Comment:
- Change currentCapacity to currentPopulation
- probably would be better to change "size" to currentPopulation, "cap" to capacity. This will help with clarity